Code/Tools/ProjectManager/Source/CreateAGemScreen.cpp

```diff
@@ -0,0 +1,446 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <AdjustableHeaderWidget.h>
+#include <DownloadController.h>
+#include <GemCatalog/GemCatalogHeaderWidget.h>
+#include <CreateAGemScreen.h>
+#include <ProjectUtils.h>
+#include <PythonBindingsInterface.h>
+
+
+#include <PythonBindingsInterface.h>
+#include <QDir>
+#include <QFileDialog>
+#include <QHBoxLayout>
+#include <QHash>
+#include <QMessageBox>
+#include <QPushButton>
+#include <QStackedWidget>
+#include <QStandardPaths>
+#include <QVBoxLayout>
+#include <QRadioButton>
+#include <QLabel>
+#include <QDialogButtonBox>
+
+
+namespace O3DE::ProjectManager
+{
+    CreateAGemScreen::CreateAGemScreen(QWidget* parent)
+        : ScreenWidget(parent)
+    {
+        QVBoxLayout* vLayout = new QVBoxLayout();
+        vLayout->setSpacing(0);
+        vLayout->setContentsMargins(0, 0, 0, 0);
+
+        QScrollArea* firstScreenScrollArea = CreateFirstScreen();
+        QScrollArea* secondScreenScrollArea = CreateSecondScreen();
+        QScrollArea* thirdScreenScrollArea = CreateThirdScreen();
+
+        connect(m_gemDisplayName->lineEdit(), &QLineEdit::textChanged, this, &CreateAGemScreen::OnGemDisplayNameUpdated);
+        connect(m_gemSystemName->lineEdit(), &QLineEdit::textChanged, this, &CreateAGemScreen::OnGemSystemNameUpdated);
+        connect(m_license->lineEdit(), &QLineEdit::textChanged, this, &CreateAGemScreen::OnLicenseNameUpdated);
+        connect(m_creatorName->lineEdit(), &QLineEdit::textChanged, this, &CreateAGemScreen::OnCreatorNameUpdated);
+        connect(m_repositoryURL->lineEdit(), &QLineEdit::textChanged, this, &CreateAGemScreen::OnRepositoryURLUpdated);
+
+        m_tabWidget = new TabWidget();
+        m_tabWidget->setObjectName("createAGemTab");
+        m_tabWidget->addTab(firstScreenScrollArea, "1. Gem Setup");
+        m_tabWidget->addTab(secondScreenScrollArea, "2. Gem Details");
+        m_tabWidget->addTab(thirdScreenScrollArea, "3. Creator Details");
```

Comment:
```suggestion
        m_tabWidget->addTab(CreateFirstScreen(), tr("1. Gem Setup"));
        m_tabWidget->addTab(CreateSecondScreen(), tr("2. Gem Details"));
        m_tabWidget->addTab(CreateThirdScreen(), tr("3. Creator Details"));
```
remove unused local vars and add translation for the tab titles.

Reply:
Resolved.